models/src_amazon_ads.yml

```diff
@@ -240,7 +240,7 @@ sources:
           - name: cost
             description: "{{ doc('cost') }}"
 
-      - name: search_term_ad_keyword_report
+      - name: search_term_ad_keyword_report 
```

Comment:
Similar to my other question in this file, I see in the source there is also a `search_term_keyword_report`. Why should we use the `ad` one instead?

Reply:
v2 vs v3 for this as well!